ENIGMAsystem/SHELL/Universal_System/Extensions/XTEST/xtest.cpp

```diff
@@ -0,0 +1,22 @@
+#include "include.h"
+
+#include "Platforms/xlib/XLIBwindow.h"
+
+namespace enigma {
+  extern unsigned short keyrmap[256];
```

Comment:
find header this shits in. if it aint in one put it somewhere.

Reply:
Done: 2a58d7a0fdb49fe8d8da94e26f018017dde971e8

But how does this affect Windows cross-compilation? Would the extension be available?